core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -59,6 +59,20 @@ oppia.directive('learnerDashboardIcons', [
                   response.data.learner_dashboard_activity_ids));
             }
           );
+          var learnerPlaylistUrl = (
+            UrlInterpolationService.interpolateUrl(
+              '/learnerplaylistactivityhandler/<activityType>/<activityId>', {
+                activityType: $scope.getActivityType(),
+                activityId: $scope.getActivityId()
+              }));
+          $http.post(learnerPlaylistUrl, {})
+            .then(function(response) {
+              if (response.data.belongs_to_subscribed_activities) {
+                $scope.canAdd = false;
```

Comment:
This variable name is too vague. Try to make it more descriptive. See the other ng-if clauses in the directive; you can tell exactly what they mean by just reading them.

Reply:
Done